DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/metadata.json

```diff
@@ -0,0 +1,86 @@
+{
+    "Slug": "edit-document",
+    "SourceFileName": "EditDocument.java",
+    "Title": "Edit Document",
+    "DescriptionLines": [
+        "**Edit a document** that is already stored in the database.<br>",
+        "You will load an existing document from the Document-Store, update it and save it back."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Loading Entities",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/session/loading-entities"
+        },
+        {
+            "Title": "The Load Command",
+            "Type" : "link",
+            "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch02/Ch02.md#load"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Load an existing document",
+            "Slug" : "step-1",
+            "DescriptionLines": [
+                "* Load an existing document from the Document Store using the Session `load()` command.<br>",
+                "",
+                "* The loaded document is added to the internal entities map (dictionary) that the Session manages.<br>",
```

Comment:
dictionary -> map. 

In Java we use maps, in c# this data structure is called Dictionary

Reply:
so will it be good just to say the following?

"The loaded document is added to the internal entities map that the Session" manages

Reply:
I think just rename Dictionary to Map.